crates/apps_lib/src/client/rpc.rs

```diff
@@ -138,7 +168,7 @@ pub async fn query_and_print_status(
 }
 
 /// Query the last committed block
-pub async fn query_block(context: &impl Namada) {
```

Comment:
I think it would probably be clearer and more flexible to return the whole `block` here

Reply:
Yeah that makes sense. Should we do it in a follow-up PR since this was merged?